### PR TITLE
Qur786/#28 migrate density data batch2

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Comment by sumitnegi7** [comment]
@qur786  create branch from main branch after syncing it so the prev commits from other branches does not show up

File: src/data/thermal-conductivity-constants.ts [hunk 1]
```diff
@@ -16,8 +16,402 @@ export const DIPPRThermalConductivityDictionary: Record<
     thermalConductivityAtMinimumTemperature: 0.2524,
     thermalConductivityAtMaximumTemperature: 0.1727,
   },
-};
 
+  Acetamide: {
```
Comment (sumitnegi7) [inline src/data/thermal-conductivity-constants.ts:20]: Again thermal conductivity data is being pushed into the PR.

Comment (qur786) [reply src/data/thermal-conductivity-constants.ts:20]: ok I will sync the local repo with remote repo and the will update  as said

Comment (sumitnegi7) [reply src/data/thermal-conductivity-constants.ts:20]: If you need help we can connect.

**Comment by qur786** [comment]
So much conflict

**Pull request closed without merging**